examples/05-custom-schema/03-font-style/Font.tsx

```diff
@@ -1,10 +1,11 @@
 import { createReactStyleSpec } from "@blocknote/react";
 
+// The Font style.
 export const Font = createReactStyleSpec(
   {
     type: "font",
     propSchema: "string",
-  },
+  } as const,
```

Comment:
I don't think these `as const` are necessary anymore, because I made the `createReactStyleSpec` contain `const` I think